src/main/java/org/folio/service/CirculationRequestsRetriever.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.service;
+
+import static org.folio.service.inventory.InventoryUtils.INVENTORY_LOOKUP_ENDPOINTS;
+import static org.folio.service.inventory.InventoryUtils.REQUESTS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.folio.rest.core.RestClient;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.core.models.RequestEntry;
+
+import io.vertx.core.Future;
+
+public class CirculationRequestsRetriever {
+
+  private static final String REQUESTS_RECORDS = "requests";
+  private static final String REQUESTS_TOTAL = "totalRecords";
+  private static final String REQUEST_ID = "id";
+
+  private final RestClient restClient;
+
+  public CirculationRequestsRetriever(RestClient restClient) {
+    this.restClient = restClient;
+  }
+
+  /**
+   * Returns number of requests for specified item.
+   *
+   * @param itemId id of Item
+   * @return future with number of requests
+   */
```

Comment:
These comments seems unhelpful, make it more informative or remove
reference to learn about comments: https://medium.com/codex/clean-code-comments-833e11a706dc
Noise Comments Section
Or Clean Code comments chapter

Reply:
Yes you are correct, the method is very straightforward, and its name speaks for it, removed unnecessary comments